### DISCUSSION TITLE
Broken: CanalLuz

### DISCUSSION THREAD

**Discussion by gorositopablo** [iptv-org/community#449] [Show and tell]
### Broken Link

https://g1.mc-slo.transport.edge-access.net/a11/ngrp:canal_luz01-100009_all/Playlist.m3u8?sense=true

### What happened to the stream?

Not loading

### Notes (optional)

This one works fine in desktop VLC (Linux in my case), but when I tried on Android VLC it fails.
This is the error message that I got in Android:

`"The location https://g1.mc-slo.transport.edge-access.net/a11/ngrp:canal_luz01-100009_all/Playlist.m3u8?sense=true cannot be played."`

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#report-a-broken-stream)

**Comment by Sphinxroot** [comment]
#EXTVLCOPT:http-referrer=https://vmf.edge-apps.net


kodi
#EXTINF:-1 ,CanalLuz master g5 adaptative
https://g5.vxral-hor.transport.edge-access.net/a11/ngrp:canal_luz01-100009_all/Playlist.m3u8?sense=true|Referer=https://vmf.edge-apps.net
#EXTINF:-1 ,CanalLuz master g3 adaptative
https://g3.vxral-slo.transport.edge-access.net/a11/ngrp:canal_luz01-100009_all/Playlist.m3u8?sense=true|Referer=https://vmf.edge-apps.net
#EXTINF:-1 ,CanalLuz 720
https://g3.vxral-slo.transport.edge-access.net/a11/ngrp:canal_luz01-100009_all/canal_luz01-100009_720p/index.m3u8|Referer=https://vmf.edge-apps.net
#EXTINF:-1 ,CanalLuz 1080
https://g5.vxral-hor.transport.edge-access.net/a11/ngrp:canal_luz01-100009_all/canal_luz01-100009_1080p/index.m3u8|Referer=https://vmf.edge-apps.net

**Comment by freearhey** [comment]
For the link to work, the player must support custom HTTP User-Agent and I guess VLC for Android does not support this feature.

https://github.com/iptv-org/iptv/blob/8adb0d0f8f993989888c2af00dfc5330728ed109/streams/ar.m3u#L210-L212
